iOS-tic-tac-toe/GameViewController.swift

```diff
@@ -79,37 +62,28 @@ public class GameViewController: UIViewController {
         return "{\"board\": \(currentBoard.description), \"gameType\": \(GameConfig.gameType)}"
     }
     
-    public func spotsEnabled(boardStackView: UIStackView, enabled: Bool) {
-        for view in boardStackView.subviews{
-            for button in view.subviews {
-                let btn = button as! UIButton
-                btn.enabled = enabled
-            }
-        }
-    }
-    
     private func completeTurn(status:String) {
         if (isGameOver){
-            endGame(boardStackView, gameLabel: playerTurnLabel, status: status)
+            endGame(status)
         } else {
-            nextTurn(boardStackView, gameLabel: playerTurnLabel)
+            nextTurn()
         }
     }
     
-    public func endGame(boardView: UIStackView, gameLabel: UILabel, status: String) {
+    public func endGame(status: String) {
```

Comment:
A couple of comments:
* `endGame` makes it sound like this function causes the game to end. Can you think of a better name?
* `status` should likely be an enum.